### ISSUE TITLE
code style & editorconfig

### ISSUE THREAD

**Issue by MithrilMan** [davidfowl/BedrockFramework#35]
@davidfowl the project has an editorconfig https://github.com/davidfowl/BedrockFramework/blob/master/.editorconfig , I think should be used to enforce some rules.

e.g. this is mine that enforce a specific use of braces, spaces, formatting and for example force you to explicit ConfigureAwait any time you await (for this a couple of analyzers are needed).

I don't mean to use my specifications, but I find this very useful to have a common style that's not dependant on the environment (VS settings) but rely on the editorconfig.

Now that different people try to contribute I think this is quite important and being you the project owner I'd say it's up to you to decide the style to use.

**Comment by davidfowl** [comment]
Agreed. I haven't figured this out yet but I'll assign this one to myself to spend some time here (I usually copy and paste a bunch of these files from previous projects).
